vocabularies/Analytics.json

```diff
@@ -161,6 +164,116 @@
+            "$Type": "Analytics.MultiLevelExpandEntry",
+            "@Core.Description": "Entries with exceptional expansion",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "SubtotalsAtBottom",
+            "$Type": "Edm.Bool",
+            "@Core.Description": "Whether to duplicate the group headers so that they appear before and after their descendants",
+            "@Core.LongDescription": "The entry before has [DrillState](Hierarchy.md#HierarchyType) `expanded`,\n            the entry after has DrillState `subtotal`.",
+            "@Core.OptionalParameter": { "DefaultValue": "false" }
+          }
+        ],
+        "$ReturnType": {
+          "$Collection": true,
+          "$Type": "Edm.EntityType",
+          "@Core.Description": "Output set including the instance annotation [`LevelInformation`](#LevelInformation)"
+        }
+      }
+    ],
+    "MultiLevelExpandLevel": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "@Core.Description": "Property names constituting a level in an [unnamed leveled hierarchy](#MultiLevelExpand)",
+      "@Core.LongDescription": "`DimensionProperties` must be used to identify entries in [`ExpandEntries/Entry`](#MultiLevelExpandEntry),\n          otherwise they have the same effect as `AdditionalProperties`.",
+      "DimensionProperties": {
+        "$Collection": true,
+        "@Core.Description": "A non-empty list of property names that define a combination of dimension values"
+      },
+      "AdditionalProperties": {
+        "$Collection": true,
+        "@Core.Description": "A possibly empty list of names of additional properties of the dimensions that occur in `DimensionProperties`"
+      }
+    },
+    "MultiLevelExpandSiblingOrder": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "@Core.Description": "Sibling order in an [unnamed leveled hierarchy](#MultiLevelExpand)",
+      "Property": { "@Core.Description": "Property by which to sort" },
+      "Descending": {
+        "$Type": "Edm.Boolean",
+        "$Nullable": true,
+        "@Core.Description": "Sort direction, ascending if not specified otherwise"
+      }
+    },
+    "MultiLevelExpandEntry": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "@Core.Description": "Expansion state of an entry in an [unnamed leveled hierarchy](#MultiLevelExpand)",
+      "Entry": {
+        "$Collection": true,
+        "@Core.Description": "An entry on a given [level](#MultiLevelExpandLevel) is identified by a list of values for the `DimensionProperties` that constitute all levels up to and including the given one",
+        "@Core.LongDescription": "The values are cast to strings as in the OData `cast` function."
+      },
+      "Levels": {
+        "$Type": "Edm.Int64",
+        "$Nullable": true,
+        "@Core.Description": "Number of levels to be expanded, null means all levels, 0 means collapsed"
```

Comment:
this is aligned with TopLevels.ExpandLevels

but 

`ShowLevels` is not aligned with `TopLevels.Levels`

, as far as I understand

Reply:
It is aligned in so far as `Levels=2` means two levels of output in both cases:
- In the recursive case, this is the root (`Sales`) plus one level (`US` and `EMEA`).
- In the leveled case, this is two grouping levels without a root.